### PR TITLE
Added CDH 632 v 1.1 app json and cluster yaml files

### PR DESCRIPTION
**Pull request by omhpe**
Added CDH 632 v 1.1 app json and cluster yaml files.

- This version adds high availability feature for namenode and resource manager. 
- The roles added for high availability are arbiter and standby.
- This also adds feature of scaling the cluster up or down which will fix the issue https://github.com/bluek8s/kubedirector/issues/318

### CODE REVIEW

File: deploy/example_catalog/cr-app-cdh632-1.1cm.json [hunk 1]
```diff
@@ -0,0 +1,421 @@
+{
+    "apiVersion": "kubedirector.hpe.com/v1beta1",
+    "kind": "KubeDirectorApp",
+    "metadata": {
+        "name" : "cdh632"
+    },
+
+    "spec" : {
+        "systemdRequired" : true,
+        "defaultPersistDirs" : ["/opt", "/var", "/data"],
```
Comment (swamibluedata) [inline deploy/example_catalog/cr-app-cdh632-1.1cm.json:10]: do we need to persist these 3 directories for all roles?

File: deploy/example_clusters/cr-cluster-cdh632-1.1cm-stor.yaml [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  # - id: arbiter
+  #   resources:
+  #     requests:
+  #       memory: "2Gi"
+  #       cpu: "2"
+  #     limits:
+  #       memory: "2Gi"
+  #       cpu: "2"
+  #   storage:
+  #     size: "50Gi"
```
Comment (swamibluedata) [inline deploy/example_clusters/cr-cluster-cdh632-1.1cm-stor.yaml:121]: remove the commented lines?

File: deploy/example_catalog/cr-app-cdh632-1.1cm.json [hunk 3]
```diff
@@ -0,0 +1,421 @@
...
+                "hive",
+                "worker"
+            ]
+        },
+        "defaultImageRepoTag": "bluedata/cdh632multi:1.1",
+        "label": {
+            "name": "CDH 6.3.2 multirole 7x",
+            "description": "CDH 6.3.2 with YARN and HBase support. Includes Hive, Hue, Impala and Spark."
+        },
+        "distroID": "bluedata/cdh632multi:1.1",
```
[the remark this answers is not in the document]
Comment (omhpe) [reply deploy/example_catalog/cr-app-cdh632-1.1cm.json]: @joel-bluedata updated distroID to bluedata/cdh632multi. Deleted newly added kdapp (cr-app-cdh632-1.1cm.json) . Kept the older file cr-app-cdh632cm.json and updated the version to 1.1.

File: deploy/example_clusters/cr-cluster-cdh632-1.1cm-stor.yaml (hunk 2 above)
Comment (omhpe) [reply deploy/example_clusters/cr-cluster-cdh632-1.1cm-stor.yaml]: @swamibluedata /data is required to be persisted only for HDFS. accordingly removed /data from defaultPersistDirs and added /data in persistDirs keys for primary, standby and arbiter roles. These roles use HDFS service.

File: deploy/example_catalog/cr-app-cdh632-1.1cm.json (hunk 1 above)
Comment (omhpe) [reply deploy/example_catalog/cr-app-cdh632-1.1cm.json]: @swamibluedata /data is required to be persisted only for HDFS. accordingly removed /data from defaultPersistDirs and added /data in persistDirs keys for primary, standby, arbiter and worker roles. These roles use HDFS service.

File: deploy/example_catalog/cr-app-cdh632cm.json [hunk 4]
```diff
@@ -120,20 +119,16 @@
                 "cmserver",
                 "primary",
                 "hive",
-                "worker",
-                "gateway",
-                "hbase",
-                "impala",
-                "apps"
```
Comment (joel-bluedata) [inline deploy/example_catalog/cr-app-cdh632cm.json:127]: Are these intentional deletions? I still see role definitions for gateway, hbase, impala, and apps.

Comment (omhpe) [reply deploy/example_catalog/cr-app-cdh632cm.json:127]: @joel-bluedata no missed that, added all the other roles definitions now.

**Pull request merged**